core/docs/d7yxc-operator_manual.md

```diff
@@ -303,3 +319,44 @@ sequenceDiagram
 (11-12) `dagger session` listens on a random available port on localhost and generates a random uuid to use as a session token (same as described in the [previous section](#dsi-basic)). The port being listened on and the session token are serialized to JSON and then written to stdout, which is a pipe connected back to the SDK parent process.
 
 (13-17) The SDK then sends GraphQL HTTP requests to the localhost listener, including the session token as a basic auth header.
+
+
+
+
+
+
+### Why we do not support rootless engine?
```

Comment:
I'm not sure we need this separate section, I think the first one is enough.

Reply:
Yeah It's pretty much the same, I'll remove it